### DISCUSSION TITLE
bdist_msi directory

### DISCUSSION THREAD

**Discussion by Holmberg-15A** [marcelotduarte/cx_Freeze#1788] [Q&A]
Is it possible that when the installer starts, it immediately starts installing to a specific directory without using the directory selection?

**Comment by marcelotduarte** [comment]
I think so, but not sure.
Are you using  [initial_target_dir](https://cx-freeze.readthedocs.io/en/latest/setup_script.html#cmdoption-arg-initial_target_dir)?
More information: https://learn.microsoft.com/en-us/windows/win32/msi/windows-installer-portal
